airflow/executors/__init__.py

```diff
@@ -4,19 +4,19 @@
 from airflow.executors.base_executor import BaseExecutor
 from airflow.executors.local_executor import LocalExecutor
 from airflow.executors.sequential_executor import SequentialExecutor
-
-try:
-    from airflow.executors.celery_executor import CeleryExecutor
-except:
-    pass
-
 from airflow.exceptions import AirflowException
 
 _EXECUTOR = configuration.get('core', 'EXECUTOR')
```

Comment:
I didn't realize until my previous comment that Airflow is enforcing capitalization for executor configuration. While you're working in this file, do you mind making that more lenient? Store `_EXECUTOR` as all lower case and do lower case comparisons thereafter.